backend/src/kitconcept/core/serializers/summary.py

```diff
@@ -7,4 +7,13 @@ class JSONSummarySerializerMetadata:
     """Additional metadata to be exposed on listings."""
 
     def default_metadata_fields(self):
-        return {"image_field", "image_scales", "effective", "Subject"}
+        return {
+            "image_field",
+            "image_scales",
+            "effective",
+            "Subject",
+            "contact_email",
+            "contact_room",
+            "contact_phone",
+            "contact_building",
```

Comment:
I don't see where these were added to the catalog metadata. That means that getting them for the summary serialization will implicitly wake up the full object.